### ISSUE TITLE
Prepare JDBF 3.0 to Maven Central release

### ISSUE THREAD

**Issue by iryndin** [iryndin/jdbf#11]
Prepare JDBF 3.0 to Maven Central release. This includes:
 - add entries to POM file
 - add javadocs
 - add javadocs and source code plugins to POM
 - add more tests
 - add detailed tutorial to the github repo

**Comment by edwgiz** [comment]
Hello Ivan,

A publishing of artefact in one of Maven repositories would be nice for most consumers,
Can I help to resolve this task?

**Comment by driv** [comment]
Hi Ivan,

I would like to help with this issue, what is the current state?

**Comment by xKopax** [comment]
this would be really nice.

**Comment by tuzzo77** [comment]
is there a jar for your project? Thanks!!

**Comment by spyhunter99** [comment]
i can help too

**Comment by spyhunter99** [comment]
@iryndin  i'm using this with osmdroid as a utility to help read metadata for esra shape files. I'd like to just use it as a dependency but since it's not published that isn't currently an option. 

 - I can publish this to central under a different group id, would you be ok with this?
 - I can roll the relevant code into our repo and link back to this one. I don't really want to maintain this though so I'd rather not do this.
 - I can help you get setup to publish this to sonatype (which syncs to central, jcenter and everything else).

Can we get this rolling? Thoughts on this?

**Comment by jobdiogenes** [comment]
One option could be. 
Use Git hub as Maven Repo.
https://gist.github.com/fernandezpablo85/03cf8b0cd2e7d8527063

**Comment by spyhunter99** [comment]
got tired of waiting, just published today's master to central as version 2.2.0, if anyone cares

**Comment by mvglasow** [comment]
Using [JitPack](https://jitpack.io/) would also work—it takes the source code of the dependency from the repo and builds it on the fly. Be aware that identifiers change:

* `groupId` is `com.github.iryndin` (if you are doing this for your own project and have your own domain, you can add a TXT record with the URL of your Github profile, then use your reverse domain name)
* `artifactId` is the name of the repo (luckily both match in this case)
* `version` must be a valid git ref, i.e. a branch, tag or hash. If you use a branch name, appending `-SNAPSHOT` will ensure you always get the latest version of that branch. So, for latest master, use `master-SNAPSHOT`. To get master as of this writing, use `bb9f01c`. (For your own project, you can add a tag matching the version name every time you release a new version.)

In a nutshell, add the following to your `<project>` (merging `<repositories>` and `<dependencies>` as needed):
```
  <repositories>
    <repository>
      <id>jitpack.io</id>
      <name>JitPack Repository</name>
      <url>https://jitpack.io</url>
    </repository>
  </repositories>
  <dependencies>
    <dependency>
      <groupId>com.github.iryndin</groupId>
      <artifactId>jdbf</artifactId>
      <version>bb9f01c</version>
    </dependency>
  </dependencies>
```
